actions/filesystem_deploy_action.go

```diff
@@ -1,9 +1,13 @@
 /*
 FilesystemDeploy Action
 
-Deploy prepared root filesystem to output image. This action requires
+Deploy prepared root filesystem to output image by copying the files from the
+temporary scratch directory to the mounted image. This action requires
 'image-partition' action to be executed before it.
```

Comment:
maybe add that it also configures optionally various files for the image (fstab etc)